### PR TITLE
Adds blog post about MVP and celerybeat

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Pulp Celerybeat and the Pulp 3.0 Minimum Viable Product (MVP)
+date: 2016-08-18T18:28:49+00:00
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: I usually let the filename identify the date only. This line can be removed.

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+layout: post
+permalink: /2016/08/18/sprint-demo-6-august-18-2016/
+author: Dennis Kliban
+tags:
+  - MVP
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: We have to specifically enable tags, we don't have one for MVP so this should be removed probably.

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - 3.0
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md:6]: We do have a tag for `3.0` so this one is good.

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  - Celerybeat
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: We have to specifically enable tags, we don't have one for Celerybeat.

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+categories:
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: We don't use categories at all this can be removed.

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  - Uncategorized
+---
+Last week during our [sprint 9 demo]({% post_url 2016-10-27-sprint-demo-9 %}) I gave an update on
+the work I did on Pulp Celerybeat for Pulp 3.0. I'd like to use this post to highlight these
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/I'd like to use this post to highlight these/This post highlights those/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+changes in written form.
+
+Pulp 3.0 is a complete rewrite of Pulp. To make sure that we could accomplish the work in a
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/make sure that we could//

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+reasonable amount of time we decided to put together a description of a [Minimum Viable Product
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/time/time,/
s/decided to//
s/of a/of the/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+(MVP) for Pulp 3.0.0](https://pulp.plan.io/projects/pulp/wiki/Pulp_3_Minimum_Viable_Product)
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/for /for the/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+release. This MVP is a subset of the functionality currently found in Pulp. We hope Pulp users will
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/currently found in Pulp/offered with Pulp 2.y/
s/We hope Pulp users will take/Please have/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+take a look at the MVP and provide feedback based on their workflows and how the MVP affects them.
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/their/your/
s/them/you/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+It is possible that we have overlooked something critical to our users, so please take a look at
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/to our users, so please take a look at MVP description/to your usage of Pulp, so please send feedback to pulp-list@redhat.com./

^ should be an email link.

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+MVP description.
+
+Looking through the MVP description, one notices that there is no mention of users being able to
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/one notices that/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+schedule any operations such as a sync or a publish. This is one the features that we decided to
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/any//

Comment (bmbouter) [reply _posts/2016-10-31-pulp-3.0-mvp.md]: I'm trying to find a way to identify nodes as a feature gap also. Maybe this paragraph could become a bulleted list of missing features? I'm OK if we don't want to do this also and keep this post focsed on just the scheduled calls.

Comment (dkliban) [reply _posts/2016-10-31-pulp-3.0-mvp.md]: I would like to make a list of all missing features, but perhaps we should make a follow up post for that.

Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/one/one of/
s/we decided to leave out of/is omitted from/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+leave out of the initial 3.0.0 release. Users will still be able to perform scheduled tasks by using
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/will still be able to perform scheduled tasks by using/can continue to schedule tasks/

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 16]
```diff
@@ -0,0 +1,23 @@
+---
+title: Pulp Celerybeat and the Pulp 3.0 Minimum Viable Product (MVP)
+layout: post
+permalink: /2016/10/31/pulp-3.0-mvp/
+author: Dennis Kliban
+tags:
+  - 3.0
+
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: Extra space is not needed.

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 17]
```diff
@@ -0,0 +1,23 @@
+---
+title: Pulp Celerybeat and the Pulp 3.0 Minimum Viable Product (MVP)
+layout: post
+permalink: /2016/10/31/pulp-3.0-mvp/
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: I usually don't include a permalink anymore

File: _posts/2016-10-31-pulp-3.0-mvp.md [hunk 18]
```diff
@@ -0,0 +1,23 @@
...
+put together a description of the [Minimum Viable Product (MVP) for the Pulp
+3.0.0](https://pulp.plan.io/projects/pulp/wiki/Pulp_3_Minimum_Viable_Product) release. This MVP is
+a subset of the functionality offered with Pulp 2.y. Please have a look at the MVP and provide
+feedback based on your workflows and how the MVP affects you. It is possible that we have
+overlooked something critical to your usage of Pulp, so please send feedback to
+[pulp-list@redhat.com](mailto:pulp-list@redhat.com).
+
+Looking through the MVP description, there is no mention of users being able to
+schedule operations such as a sync or a publish. This is one of the features that is omitted from
+the initial 3.0.0 release. Users can continue to schedule tasks by using other tools such as cron.
```
Comment (bmbouter) [inline _posts/2016-10-31-pulp-3.0-mvp.md]: s/by //

**Review by bmbouter — APPROVED** [review]
A few small changes suggested. The post content looks great, and I verified all the links and content render nicely. Thanks @dkliban

**Pull request merged**